keep/rulesengine/rulesengine.py

```diff
@@ -156,6 +157,23 @@ def _run_cel_rules(
 
                                 send_created_event = incident.is_visible
 
+                            incident_id = incident
```

Comment:
that's also quite confusing, what does `incident_id = incident` mean?

Reply:
it should be incident_id = incident.id

Fixed